synapseAnnotations/data/ngs.json

```diff
@@ -38,19 +38,7 @@
     "name": "isStranded",
     "description": "Whether or not the library is stranded.",
     "columnType": "BOOLEAN",
-    "maximumSize": 250,
-    "enumValues": [
-      {
-        "value": true,
-        "description": "",
-        "source": ""
-      },
-      {
-        "value": false,
-        "description": "",
-        "source": ""
-      }
-    ]
+    "maximumSize": 250
```

Comment:
```suggestion
```

Think this got in there extra?

Reply:
I think it's just showing up as a changed line because I removed the comma. There is still only one instance of `maximumSize`.

Reply:
Might be easier to see in the side-by-side view

<img width="1032" alt="Screen Shot 2019-12-11 at 11 10 06 AM" src="https://user-images.githubusercontent.com/4452678/70652070-d52b7d00-1c06-11ea-8bb2-b36d0c8f3395.png">